<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emprestimo</class>
 <widget class="QWidget" name="Emprestimo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>553</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>553</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Emprestimo</string>
  </property>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>130</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="btnCarregar">
          <property name="text">
           <string>Carregar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Data da curva:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDateEdit" name="deditDataCurva"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblDownload">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tblCalculo">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>Data</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Taxa</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Valor:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txtValor"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnCalcular">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Inerpolar data para taxa: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDateEdit" name="deditDataInterpolacao"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;T&lt;/p&gt;&lt;p&gt;A&lt;/p&gt;&lt;p&gt;X&lt;/p&gt;&lt;p&gt;A&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>                        Número de dias utéis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="graphics"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>btnSair</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSair</sender>
   <signal>clicked()</signal>
   <receiver>Emprestimo</receiver>
   <slot>fechar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalcular</sender>
   <signal>clicked()</signal>
   <receiver>Emprestimo</receiver>
   <slot>interpolar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCarregar</sender>
   <signal>clicked()</signal>
   <receiver>Emprestimo</receiver>
   <slot>carregar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fechar()</slot>
  <slot>interpolar()</slot>
  <slot>carregar()</slot>
 </slots>
</ui>
